src/unwinder.rs

```diff
@@ -611,6 +635,24 @@ impl<D: Deref<Target = [u8]>> ModuleUnwindDataInternal<D> {
                 base_addresses: base_addresses_for_sections(section_info),
                 text_data,
             }
+        } else if let Some(pdata) = section_info.section_data(b".pdata") {
+            let mut range_and_data = |name| {
+                section_info
+                    .section_svma_range(name)
+                    .and_then(|range| {
+                        Some(Range {
+                            start: (range.start - section_info.base_svma()).try_into().ok()?,
+                            end: (range.end - section_info.base_svma()).try_into().ok()?,
+                        })
+                    })
+                    .zip(section_info.section_data(name))
```

Comment:
I find the use of zip to create the tuple a bit obscure. I'd prefer `let range = ...?; let data = ...?; Some((range, data))` . That last line could then be `Some(DataAtRvaRange::new(range, data))` if you decide to go with that.